database/seeder.js

```diff
@@ -88,6 +88,6 @@ const seed = async () => {
     }
 
   });
-}/**********************************/
+};/**********************************/
```

Comment:
I think having lines like `/********/` might be a decent indicator that code in that section could be refactored into its own file.